Build/Customizations/Projects.props

```diff
@@ -64,6 +64,10 @@
       <TestConfiguration>$(NormalTestConfiguration)</TestConfiguration>
     </UnitTestProjectFiles>
 
+    <UnitTestProjectFiles Include="$(SolutionDirectory)UnitTests.NUnit2\UnitTests.NUnit2.csproj">
```

Comment:
@MichaelKetting  Makes sense, are you sure about **Integration**Tests.MediumTrust instead of **Unit**Tests.MediumTrust, I know it would more correct, but it breaks the symmetry of the projects.

Reply:
@iam-mholle I was struglging with this, too and only changed it to IntegrationTests during the final round of tweaking the CR. The tests we're exercising are located in the integrationtests subnamespace within the unitests project and we're testing that MediumTrust (and integrative feature) works for those tests. Only downside: linq sqlbackend up until now used integrationtests only for DB tests. Does the reasoning pro IntegrationTests as the name make sense or does it stil feel iffy? Or rather, more iffy than using UnitTests here.

Reply:
No, it makes sense 😊